subroutines.sh

```diff
@@ -35,9 +35,23 @@ replace_value_at_position() {
 # Function
 check_presence() {
   TYPE="$1" # BLE MAC, LN
```

Comment:
I don't really see where TYPE is necessary other than logging?

Reply:
True - that is indeed useless today. Will remove it, thank you - again you have good eyes and definitely going in the code :)

The history... in the first iteration of check_presence ( ~ 2 weeks ago) the function was receiving the Type and a pattern Match. The pattern could a MAC, BLE LN or both. Thus the reason to pass the type.

By adding in this PR a default Mac address `FF:FF....:FF` , there's a defined Mac address (valid or not) for each VIN/BLE LN that can be pass to the function. So... Match=$2 is now replaced by BLE LN=$2  * MAC addr=$3.